kebab/src/wrapper.ts

```diff
@@ -14,6 +14,8 @@ export const wrap = (connection: Connection) => ({
   query: {
     getTopPublicRooms: (cursor = 0): Promise<GetTopPublicRoomsResponse> =>
       connection.fetch("get_top_public_rooms", { cursor }),
+    getScheduledRooms: (cursor = "", getOnlyMyScheduledRooms = false): Promise<GetScheduledRoomsResponse> =>
```

Comment:
the type of `cursor` is probably `number | ""`

Reply:
Fixed